app/views/summary/arrival_summary_page.scala.html

```diff
@@ -80,33 +80,15 @@
                         ),
                         value = Value(
                             content = Text(data.arrivalDetails.map(_.dateOfArrival.toString).getOrElse(""))
-                        ),
-                        actions = Some(Actions(
-                            items = Seq(
-                                ActionItem(
-                                    href = s"${controllers.movements.routes.MovementDetailsController.displayPage}",
-                                    content = Text(messages("site.change")),
-                                    visuallyHiddenText = Some(messages("summary.arrival.date.change"))
-                                )
-                            )
-                        ))
+                        )
```

Comment:
Just curious; why no change links?

Reply:
Because they are not present on other summary pages. Wasn't there a ticket to remove those?
Arrival Summary page haven't had view tests for some time, so it might have been omitted.

Reply:
Are you sure?  I've just tried Arrival in internal and external service.  The date/time and location have change links

Reply:
Spoke to the BAs about t and you're right. So there was an error on Arrival Summary page, that 'change' links were not present while they should. fixing this just now.